app/assets/stylesheets/main.scss

```diff
@@ -1373,13 +1373,11 @@ fieldset[disabled] {
   border: none;
 }
 
-.notice {
-    color: darkgreen;
-}
-.alert {
-    color: red;
+.alert{
```

Comment:
@igorkasyanchuk, this classes already defined in bootstrap css, so we didn't need it here. 
Also, it makes all flash text red (even if we have success or notice messages, it is not OK